metaflow/plugins/argo/argo_workflows_cli.py

```diff
@@ -594,3 +594,37 @@ def _convert_value(param):
             "See the run in the UI at %s" % run_url,
             bold=True,
         )
+
+
+@argo_workflows.command(help="Suspend flow execution on Argo Workflows.")
+@click.option(
+    "--authorize",
+    default=None,
+    type=str,
+    help="Authorize the suspension with a production token",
+)
+@click.argument("run-id", required=True, type=str)
+@click.pass_obj
+def suspend(obj, run_id, authorize=None):
+    # TODO: implement production token authorize
```

Comment:
Let's implement this before shipping. The logic can be the same as the one that is in-play for deploys